src/components/organisms/AppBar/AppBar.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import {
+  AppBar,
+  Box,
+  Container,
+  IconButton,
+  ListItemIcon as ListIconItem,
+  Menu,
+  MenuItem,
+  MenuList,
+  Toolbar,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+
+import { NextLinkAnchor } from 'components/atoms';
+
+import { Menu as MenuIcon } from '@mui/icons-material';
+import {
+  AboutIcon,
+  FridgeAddIcon,
+  FridgeFindIcon,
+  GuidelineIcon,
+  HomeIcon,
+  LogoIcon,
+  VolunteerIcon,
+} from 'theme/icons';
+
+const menu = [
+  { icon: HomeIcon, title: 'Home', link: '/' },
+  { icon: FridgeFindIcon, title: 'Find a Fridge', link: '/fridge/find' },
+  { icon: FridgeAddIcon, title: 'Add a Fridge', link: '/fridge/add' },
+  { icon: AboutIcon, title: 'About', link: '/about' },
+  { icon: GuidelineIcon, title: 'Best Practices', link: '/guideline' },
+  { icon: VolunteerIcon, title: 'Volunteer', link: '/volunteer' },
+];
+
+export default function ResponsiveAppBar() {
+  const [anchorElNav, setAnchorElNav] = React.useState(null);
+
+  const handleOpenMenuMobile = (event) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseMenuMobile = () => {
+    setAnchorElNav(null);
+  };
+
+  const MenuDesktop = () =>
+    menu.slice(1).map((item) => (
+      <Tooltip title={item.title} key={item.title}>
+        <IconButton
+          aria-label={item.title}
+          component={NextLinkAnchor}
+          to={item.link}
+        >
+          <item.icon sx={{ borderRadius: '50%' }} />
+        </IconButton>
+      </Tooltip>
+    ));
+
+  const MenuMobile = () => (
+    <MenuList>
+      {menu.map((item) => (
+        <MenuItem key={item.title} onClick={handleCloseMenuMobile}>
+          <ListIconItem>
+            <item.icon sx={{ borderRadius: '50%' }} />
+          </ListIconItem>
+          <Typography
+            aria-label={item.title}
+            component={NextLinkAnchor}
+            to={item.link}
+            sx={{ color: 'inherit', textDecoration: 'none' }}
+          >
+            {item.title}
+          </Typography>
+        </MenuItem>
+      ))}
+    </MenuList>
+  );
+
+  return (
+    <AppBar position="sticky">
+      <Container maxWidth="xl">
+        <Toolbar>
+          <LogoIcon />
+          <Typography
+            noWrap
+            aria-label="Home"
+            component={NextLinkAnchor}
+            to="/"
+            sx={{
+              flexGrow: { xs: 1, md: 0 },
+              mr: 2,
+              display: 'flex',
+              fontFamily: 'MuseoModerno',
+              fontWeight: 600,
+              color: 'inherit',
+              textDecoration: 'none',
+            }}
+          >
+            Community Fridge Map
```

Comment:
Color is incorrect: Should be version of black
Also "Community Fridge Map" is supposed to be multi-line

<img width="242" alt="Screen Shot 2022-06-21 at 11 07 46 PM" src="https://user-images.githubusercontent.com/104579341/174934998-3af128f7-f942-48fe-9003-3d62f9d34aef.png">
e